smoketest/scripts/cli/test_interfaces_bridge.py

```diff
@@ -45,17 +45,33 @@ def setUp(self):
             for tmp in Section.interfaces("ethernet"):
                 if not '.' in tmp:
                     self._members.append(tmp)
+            
+            self.session.set(['interfaces','dummy','dum0'])
+            self.session.set(['interfaces','dummy','dum1'])
+            self.session.commit()
+            self.session.set(['interfaces','bonding','bond1','member','interface','dum0'])
+            self.session.set(['interfaces','bonding','bond1','member','interface','dum1'])
+            self.session.commit()
```

Comment:
Why is this `commit()` required?

Reply:
Mainly to make the required interface effective during testing

Reply:
This `commit` must exist, otherwise the following code will not take effect:

https://github.com/jack9603301/vyos-1x/blob/b128f4e938e671af31907f154945876c60b0db72/smoketest/scripts/cli/test_interfaces_bridge.py#L54-L56

Reply:
`setUp()` is called for every testcase and within every testcase we call `commit()`, so calling `commit` here is redundant, please remove.

See https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp for more info.